code/game/objects/effects/aliens.dm

```diff
@@ -373,9 +372,17 @@
 	desc = "Burbling corrossive stuff. The radical kind."
 	icon_state = "acid-hyper"
 
-/obj/effect/alien/acid/New(loc, target)
+/datum/locking_category/acid
+
+/obj/effect/alien/acid/New(loc, atom/target)
 	..(loc)
 	src.target = target
+	pixel_x = target.pixel_x
+	pixel_y = target.pixel_y
+	if(istype(target,/atom/movable))
+		var/atom/movable/locker = target
+		locker.lock_atom(src, /datum/locking_category/acid)
+		glide_size = locker.glide_size
```

Comment:
Not really looking to rewrite atom locking code on top of all of this, and I don't think its a good idea to add it in just as a general thing without more thought